File: apps/sepuling/src/test/java/smecalculus/bezmen/storage/SepulkaDaoMyBatisPostgresIT.java
package smecalculus.bezmen.storage;

import org.springframework.test.context.ContextConfiguration;
import smecalculus.bezmen.construction.SepulkaDaoBeans;
import smecalculus.bezmen.construction.StoragePropsBeans;

@ContextConfiguration(classes = {StoragePropsBeans.MyBatisPostgres.class, SepulkaDaoBeans.MyBatis.class})
public class SepulkaDaoMyBatisPostgresIT extends SepulkaDaoIT {}

File: apps/sepuling-kotlin/src/test/kotlin/smecalculus/bezmen/storage/SepulkaDaoMyBatisPostgresIT.kt
package smecalculus.bezmen.storage

import org.springframework.test.context.ContextConfiguration
import smecalculus.bezmen.construction.SepulkaDaoBeans
import smecalculus.bezmen.construction.StoragePropsBeans

@ContextConfiguration(classes = [StoragePropsBeans.MyBatisPostgres::class, SepulkaDaoBeans.MyBatis::class])
class SepulkaDaoMyBatisPostgresIT : SepulkaDaoIT()
